depends/README.md

```diff
@@ -95,6 +95,7 @@ The following can be set when running make: make FOO=bar
     DEBUG: disable some optimizations and enable more runtime checking
     HOST_ID_SALT: Optional salt to use when generating host package ids
     BUILD_ID_SALT: Optional salt to use when generating build package ids
+    ALLOW_HOST_PACKAGE: compile using hosts library set (via pkg-config)
```

Comment:
Concept ACK.

@skmcontrib 
Could you emphasize that the host's packages are used when they are not built due to the `NO_*` variable set?

Also this line could be moved up to be just under the `NO_UPNP` line.

Reply:
Can you please elaborate a bit further what this means? What is compiled using hosts libraries? And which host libraries, which packages are replaced by host packages?